ONSdigital/census-rm-performance-tests
# Add scenario label field to JSON logs

## Context
We need something in the logs to identify the scenario running

## What has changed
* Add `scenario_tag` field to json metric logs
* Remove "skip" tag logic and documentation (It's just as easy to comment out a scenario as to edit a tag into it)
* Set the scenario tag in the context in the before scenario hook

## How to test
Run a performance test and check the log outputs have the `scenario_label` field correctly set (you may need to add `--no-capture` to your behave command to see logs).

## Links
https://trello.com/c/6GIhpjIV/682-add-performance-test-scenario-labels-to-metrics-5
